lib/Target/OpenQASM/TranslateToOpenQASM.cpp

```diff
@@ -207,8 +207,13 @@ static LogicalResult emitOperation(Emitter &emitter, func::FuncOp op) {
 }
 
 static LogicalResult emitOperation(Emitter &emitter, quake::ExtractRefOp op) {
+  std::optional<int64_t> index = std::nullopt;
```

Comment:
```suggestion
  std::optional<int64_t> index;
```
This is take or leave as you wish. But for brevity, one can omit the `std::nullopt` being fully specified.